README.md

```diff
@@ -99,32 +99,40 @@ ansible-playbook -i hosts -v deploy-okd.yml --extra-vars "compliance_operator=tr
 Once the playbook tell you to, boot the masters on
 Fedora CoreOS USB and start the installation process. 
 The NUCs are a bit slow on the network side so a number of kernel arguments are needed.
+
+```shell
+# Automated version
```

Comment:
Perhaps a different comment like "Use hostname from DHCP or DNS" is more appropriate?